autora/cycle/_simple_cycle.py

```diff
@@ -26,26 +27,85 @@ class _SimpleCycleData:
     theories: List[BaseEstimator]
 
 
-class _SimpleCycle:
+def _get_cycle_properties(data: SimpleCycleData):
     """
+    Examples:
+        Even with an empty data object, we can initialize the dictionary,
+        >>> cycle_properties = _get_cycle_properties(SimpleCycleData(metadata=VariableCollection(),
+        ...     conditions=[], observations=[], theories=[]))
+
+        ... but it will raise an exception if a value isn't yet available when we try to use it
+        >>> cycle_properties["%theories[-1]%"] # doctest: +ELLIPSIS
+        Traceback (most recent call last):
+        ...
+        IndexError: list index out of range
+
+        Nevertheless, we can iterate through its keys no problem:
+        >>> [key for key in cycle_properties.keys()] # doctest: +NORMALIZE_WHITESPACE
+        ['%observations.ivs[-1]%', '%observations.dvs[-1]%', '%observations.ivs%',
+        '%observations.dvs%', '%theories[-1]%', '%theories%']
+
+    """
+
+    n_ivs = len(data.metadata.independent_variables)
+    n_dvs = len(data.metadata.dependent_variables)
+    cycle_property_dict = LazyDict(
+        {
+            "%observations.ivs[-1]%": lambda: data.observations[-1][:, 0:n_ivs],
+            "%observations.dvs[-1]%": lambda: data.observations[-1][:, n_ivs:],
```

Comment:
guessing i'm not parsing this quite right, but just wanted to confirm: should be `n_ivs:` and not `n_dvs:`? 

same question with line 59 below, too.